src/Documentation/implementation/streams_implementation/azure_queue_streams.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Azure Queue Streams Implementation Details
+---
+
+# Orleans Azure Queue Streams Implementation Details
+
+Each stream provider (Azure Queues, EventHub, SMS, SQS, ...) has it's own queue specific details and configuration.
+This section provides some details about the usage, configuration and implementation of **Orleans Azure Queue Streams**.
+More details are available in the streaming tests, which contain most of the configuration options, specifically [**`AQClientStreamTests`**](https://github.com/dotnet/orleans/tree/master/test/Extensions/TesterAzureUtils/Streaming/AQClientStreamTests.cs), [**`AQSubscriptionMultiplicityTests`**](https://github.com/dotnet/orleans/tree/master/test/Extensions/TesterAzureUtils/Streaming/AQSubscriptionMultiplicityTests.cs), and the extension functions for [**`IAzureQueueStreamConfigurator`**](https://github.com/dotnet/orleans/tree/master/src/Azure/Orleans.Streaming.AzureStorage/Providers/Streams/AzureQueue/AzureQueueStreamBuilder.cs)  and [**`ISiloPersistentStreamConfigurator`**](https://github.com/dotnet/orleans/tree/master/src/Orleans.Runtime.Abstractions/Streams/ISiloPersistentStreamConfigurator.cs).
```

Comment:
> "More details are available in the streaming tests, which contain most of the configuration options, specifically"

- Let's make sure readers understand that this section is not comprehensive.  Maybe something like:

"This section is not comprehensive, and more details are available in the streaming tests, which ..."

Reply:
try now